src/libraries/System.Private.Xml/tests/XmlWriter/DisposeTests.cs

```diff
@@ -177,10 +176,13 @@ await using (var asyncWriter = XmlWriter.Create(stream2, new XmlWriterSettings()
                     syncWriter.WriteEndElement();
                 }
 
+                Assert.Equal(@"?<?xml version=""1.0"" encoding=""utf - 8""?><root><test abc=""1"" /></root>", Encoding.UTF8.GetString(stream1.GetBuffer()));
```

Comment:
what's the question mark in front?

Reply:
Apparently BOM, updated the test to not emit BOM and using UTF8Encoding instead Encoding.Utf8 for parsing the string